### ISSUE TITLE
Left/right icons on Samsung Galaxy S7

### ISSUE THREAD

**Issue by yochees** [AnySoftKeyboard/AnySoftKeyboard#786]
One or two releases ago left/right icons chaged from flat white, to what is in screenshot:
https://goo.gl/photos/Hznaefnk1phnCmWz5

Oddly enough on Xperia XZ running same Android 6 they are OK.

**Comment by menny** [comment]
ya... I  don't think there's much I can do about this. Unless we add the arrows to the theme specifications.

**Comment by yochees** [comment]
Yes we can! ;) At least I hope so. 

While this is Samsung specific issue, it arised after ASK update (as opposed to Samsung update). I was using beta and the arrows got wrong after ASK update. So I've switched off beta, got older release version and it was OK. Until ASK release version was updated. So my hope is that if it was broken with some ASK update, the bug can be traced and fixed. If there is a way to get older version, I can pinpoint the version where things turned wrong. 

One more screenshot here. 
![screenshot_20161221-111828](https://cloud.githubusercontent.com/assets/24437104/21390169/2205036a-c78e-11e6-8731-f4809ed476db.png)
